data_prep/project_src.py

```diff
@@ -125,7 +125,7 @@ def _get_harness(src_file: str, out: str, language: str) -> tuple[str, str]:
 
   content = _format_source(src_file)
 
-  if language == 'c++' and 'int LLVMFuzzerTestOneInput' not in content:
+  if language in {'c++', 'c'} and 'int LLVMFuzzerTestOneInput' not in content:
```

Comment:
`language. lower()` would be more robust.
Just in case the language string [is capitalized](https://github.com/google/oss-fuzz-gen/blob/e5a4fbff52496dd157dc967e1af11c7f08837499/experiment/oss_fuzz_checkout.py#L151).

Reply:
done, thanks